src/hotspot/share/runtime/lightweightSynchronizer.cpp

```diff
@@ -60,14 +60,14 @@ class ObjectMonitorTable : AllStatic {
     }
     static void* allocate_node(void* context, size_t size, Value const& value) {
       ObjectMonitorTable::inc_items_count();
-      return AllocateHeap(size, MEMFLAGS::mtObjectMonitor);
+      return AllocateHeap(size, MemTag::mtObjectMonitor);
```

Comment:
pre-existing: Why the scope here and below?

Reply:
Good question, I will clean this up.